### PR TITLE
Player: Add lookingAt, Add offhand option for getHeldItem

### PR DESCRIPTION
**Pull request by camnwalter**

### CODE REVIEW

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt [hunk 1]
```diff
@@ -222,26 +229,49 @@ object Player : CTWrapper<ClientPlayerEntity?> {
...
+     * Gets the current item in the player's hand.
      *
+     * @param mainHand whether to get the item in the main hand or offhand
      * @return the current held [Item]
      */
+    @JvmOverloads
     @JvmStatic
-    fun getHeldItem(): Item? = toMC()?.inventory?.selectedSlot?.let {
-        getInventory()?.getStackInSlot(it)
+    fun getHeldItem(mainHand: Boolean = true): Item? {
```
Comment (mattco98) [inline src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt:265]: Might be nice to add an overload that accepts `Hand` instead, which would pair nicely with the recently changed you made to the interaction trigger

**Review by mattco98 — APPROVED** [review]

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt (hunk 1 above)
Comment (camnwalter) [reply src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt:265]: Should I add Hand into providedlibs then?

Comment (mattco98) [reply src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt:265]: Yeah I guess that makes sense. I was hesitant about putting more stuff in the global scope, but it looks like Rhino will let you override global declarations with `var` so it's fine

**Comment by camnwalter** [comment]
Added a breaking change in LivingEntity. Can remove it if you would like

**Pull request merged**